cmd/main.go

```diff
@@ -68,6 +73,19 @@ const (
 	keySize               = 16
 )
 
+// LogHook to send a graphite metric for each log entry
+func (h LogHook) Run(e *zerolog.Event, level zerolog.Level, msg string) {
+	metrics, err := h.config.metrics()
+	if err != nil {
+		return
+	}
+	h.metric = metrics
+	h.metric.RecordByLogTypes(level.String())
+	if err = h.metric.Init(); err != nil {
```

Comment:
Do I understand correctly it will call Init() on metrics every time we send logs?
If so, it is not good because on every log line we will run own metrics goroutine and discard one we already assigned, it's resource consuming and probably not correct